components/patient-data/impl/src/test/java/org/phenotips/data/internal/controller/AbstractComplexControllerTest.java

```diff
@@ -0,0 +1,428 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see http://www.gnu.org/licenses/
+ */
+package org.phenotips.data.internal.controller;
+
+import com.xpn.xwiki.objects.BaseProperty;
```

Comment:
Are these imports in the codestyle order? Would have expected this `com.xpn.xwiki` import to be with the others below.